src/renderer/pages/Staking/Overview/Overview.tsx

```diff
@@ -112,7 +114,7 @@ export const Overview = () => {
     const isWalletConnect = walletUtils.isWalletConnect(activeWallet);
     const isPolkadotVault = walletUtils.isPolkadotVault(activeWallet);
 
-    if (isMultisig || isNovaWallet || isWalletConnect || isPolkadotVault) {
+    if (isMultisig || isNovaWallet || isWalletConnect || (isPolkadotVault && addresses.length < 2)) {
```

Comment:
`=== 1` is a more straightforward condition, `< 2` makes me wonder what can go wrong with 2 address
```suggestion
    if (isMultisig || isNovaWallet || isWalletConnect || (isPolkadotVault && addresses.length === 1)) {
```